From 6069e47ee6cb30197390dc58f96487fc4cd47753 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Sat, 12 Jul 2025 21:48:59 -0400
Subject: [PATCH] elf.h: add AT_HWCAP3 and AT_HWCAP4

there are probably more new auxv keys that should be added, but these
are added now specifically because we may need to mask them.
---
 include/elf.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/elf.h b/include/elf.h
index 2ce88353..87e9f471 100644
--- a/include/elf.h
+++ b/include/elf.h
@@ -1040,6 +1040,8 @@ typedef struct {
 #define AT_RANDOM	25
 
 #define AT_HWCAP2	26
+#define AT_HWCAP3	29
+#define AT_HWCAP4	30
 
 #define AT_EXECFN	31
 
-- 
2.34.1

